akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -2854,9 +2854,33 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    * by the monitor unchanged. Note that the monitor inserts a memory barrier every time it processes an
    * event, and may therefor affect performance.
    * The `combine` function is used to combine the `FlowMonitor` with this flow's materialized value.
+   *
+   * @deprecated Use monitor() or monitorMat(combine) instead
    */
+  @Deprecated
+  @deprecated("2.5.12", "Use monitor() or monitorMat(combine) instead")
```

Comment:
we discussed this elsewhere, I think you are overdoing the deprecations, now 3?

The JavaDoc is added automatically, see https://doc.akka.io/japi/akka/current/akka/actor/ActorIdentity.html#getRef--

Reply:
Thanks, seems the scaladoc one can be omitted then -- we have this duplication in some older classes as well which I've seen I guess; Will make a ticket about those separately